config/core/test-configs.yaml

```diff
@@ -2410,6 +2410,7 @@ test_configs:
       - kselftest-arm64
       - kselftest-cpufreq
       - kselftest-futex
+      - kselftest-kvm
```

Comment:
OK with gcc-10 in `lab-broonie`: https://staging.kernelci.org/test/plan/id/62b80820ec948a2ab574f1e8/
A couple of test cases were skipped or failed but the tests were run correctly afaict.

The tests were not run with `clang-15` however: https://staging.kernelci.org/test/plan/id/62b7fac553049e8b7874f1d8/
https://storage.staging.kernelci.org/next/master/next-20220624/arm64/defconfig/clang-15/lab-broonie/kselftest-kvm-meson-gxl-s905x-libretech-cc.html#L1581
```
No such collection 'kvm'
```
because the kvm kselftest build failed:
https://storage.staging.kernelci.org/next/master/next-20220624/arm64/defconfig/clang-15/logs/kselftest.log
```
In file included from include/kvm_util.h:10:
include/kvm_util_base.h:443:14: warning: declaration of 'struct kvm_nested_state' will not be visible outside of this function [-Wvisibility]
                                         struct kvm_nested_state *state)
                                                ^
include/kvm_util_base.h:445:19: error: invalid application of 'sizeof' to an incomplete type 'struct kvm_nested_state'
```

@montjoie It's failing to boot in `lab-baylibre`: https://staging.kernelci.org/test/plan/id/62b7f56e116d64a65d74f1ec/

Reply:
The links above were from linux-next, there's a build issue there for the kvm kselftest but maybe mainline works. We can check that again with some staging-mainline results.

Reply:
clang kselftest builds remain broken for any cross compilation.

Reply:
OK so we can just ignore that as a known issue as far as this PR is concerned.

@montjoie It would still be good to understand why it's not booting in BayLibre's lab.